### PR TITLE
Feature/update phpunit

### PR DESCRIPTION
**Pull request by mtompset**
vagrant up
vagrant ssh
cd /var/lib/GA<tab completion>
make test

### CODE REVIEW

File: vagrant-provision.sh [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
-# Install Apache and PHP (and any needed extensions).
-sudo yum install -y git php php-mcrypt php-pdo php-xml php-mbstring
+# Make sure apt has current list/updates
+sudo apt update
+sudo apt upgrade -y
+
+# Install necessary PHP building blocks
+# Install Apache and PHP (and any needed extensions). 
+# Install mock DB stuff
+sudo apt install -y zip unzip php php-dev php-pear \
```
Comment (forevermatt) [inline vagrant-provision.sh]: Why the need for `php-pear` and `php-dev`?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,2 @@
+test:
+	cd /var/lib/GA_mock/SilMock/tests; /var/lib/GA_mock/SilMock/tests/phpunit
```
Comment (forevermatt) [inline Makefile]: These would commonly be on two separate lines:
```suggestion
	cd /var/lib/GA_mock/SilMock/tests
	/var/lib/GA_mock/SilMock/tests/phpunit
```

Comment (mtompset) [reply Makefile]: > These would commonly be on two separate lines:

wouldn't be in the correct directory if separate lines.

File: vagrant-provision.sh (hunk 1 above)
Comment (mtompset) [reply vagrant-provision.sh]: Removed as suggested.

**Pull request merged**